docs/release-notes/april-2024.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: April-2024
+hide_title: true
+title: April 2024
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for April 2024
+
+## 🌐 Semgrep AppSec Platform
+
+<!-- Sara -->
+
+### Added
+
+- The **Teams** feature is now in public beta. This feature enables you to assign members to teams, and then grant those teams access to specific projects (repositories added to Semgrep).
+    - Teams are crucial to large organizations with hundreds of members and projects. See [<i class="fa-regular fa-file-lines"></i> Manage user access to projects](/deployment/team).
+- The Dashboard now displays the Assistant **priority inbox**, a list of essential tasks that Semgrep Assistant prepares for you each time you log in. <!-- 13768 -->
+- CLI token activation page <!--13660 -->
+
+### Changed
+
+- **Editor and playground**: Structure mode has replaced simple mode. Try it out in the [<i class="fas fa-external-link fa-xs"></i> Playground](https://semgrep.dev/playground/new).
+- Semgrep Cloud Platform has been renamed to Semgrep AppSec Platform.
+- The Dashboard now has several UX improvements.
+- The default Bitbucket YAML configuration file has been updated with options for full, diff, and on-demand scans.
+- Improved the process of creating a GitHub Enterprise private Semgrep app. <!-- 13675 -->
+- **Settings**: The Semgrep Pro Engine toggle has been renamed to <i class="fa-solid fa-toggle-large-on"></i> Cross-file.
+
+### Fixed
+
+## 💻 Code
+
+### Added
+
+- Added support for the QL language, which is used by CodeQL.
+- Added the ability to specify multiple output flags, which allows users to write output to multiple files in multiple formats.
+- Added the ability to copy autofix suggestions displayed on the **Findings** page.
+- Added the ability to filter findings generated by Pro rules on the **Findings** page.
+
+### Changed
+
+- Cross-file (intrafile) analysis is now the default for Semgrep Code.
+- Switched regex engines; some rules may need to be updated, since Semgrep's new regex engine is stricter.
+
+### Fixed
+
+- Fixed the SARIF output obtained from the CLI so that it includes dataflow traces.
+- Semgrep no longer uses the `LOG_LEVEL` and `PYTEST_LOG_LEVEL` environment variables to determine the log level. Semgrep only considers `SEMGREP_LOG_LEVEL`, as well as `PYTEST_SEMGREP_LOG_LEVEL` in Semgrep's Pytest tests.
```

Comment:
Don't know if this is front-facing - I checked our environment variables in https://semgrep.dev/docs/cli-reference and it doesn't mention these. I also checked the 1.71.0 CLI additions; still nothing.